tc/graphene/python_worker/avalon_worker/base_work_order_processor.py

```diff
@@ -275,31 +292,23 @@ def _get_mrenclave(self):
             mrenclave value of worker enclave as hex string.
             If Intel SGX environment is not present returns empty string.
         """
+        logger.info("GET mr enclave")
         mrenclave = self.sgx_attestation.get_mrenclave()
+        logger.info(
```

Comment:
Debug log!

Reply:
Added while debugging functionality. Not really useful, removed it